docs/contributor-guide/developer-guide/testing.md

```diff
@@ -181,6 +181,14 @@ To run the tests:
     bin/run-browser-tests landing_page.test.ts
     ```
     Use the `--debug` flag to print debug logs as the test runs.
+    
+    
+<b>TIP:</b> To speed up the running of tests beyond selecting a specific test file,
+you can run an individual test case. Open the test file and replace the line that 
+begins with "it" 
+```
+it("", async => {})   replace with    it("", async => {})
```

Comment:
this should be `fit` correct?

This also works with describe too for a sub-set of tests `fdescribe`

Reply:
Done